docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
+
+Sets the nodes next boot device persistently
+
+device <dev> can be set to:
+
+  none  : Do not change boot device order
+  
+  pxe   : Force PXE boot
```

Comment:
The word "force" really doesn't need to be here; you're just changing a setting, there's not really any kind of "hard override" semantics here. And of course it's redundant with every other item in the list (except the one I suggested removing).